crypto/nrf_cc310_platform/include/nrf_cc3xx_platform_identity_key.h

```diff
@@ -24,8 +24,9 @@
  *
  * @note    This API assumes no format of the data stored.
  *
- * @note    The key material stored using this API should be kept secret, for instance
- *          by generating it directly or the device or securely provisioning it.
```

Comment:
Further improvements are needed.

What are we trying to say when we say

"by generating it directly or the device or securely provisioning it."

Reply:
should it be "on the device" ?

Reply:
It should be on the device, yes. Seems like a typo